doc/integrating_applications/topics/adding_dropbox_connection_finish.adoc

```diff
@@ -0,0 +1,23 @@
+[id='adding-dropbox-connection-finish']
+= Adding a Dropbox connection as a finish connection
+
+In an integration, to add a Dropbox connection as the finish connection,
+start creating the integration, add and configure the start connection,
+and then:
+
+. On the *Choose a Finish Connection* page, click the Dropbox connection that
+you want to use as the integration's finish connection. 
+. On the *Choose an Action* page, click the *Upload a file to Dropbox* 
+action to add the current integration data to the
+Dropbox account that this connection accesses. 
+. In the *Remote Path* field, enter the path of the Dropbox folder 
+in which to store the integration data. 
+. For the *Upload mode* select:
++
+** *Add* to rename a file that you are uploading if a file with the
```

Comment:
In this case if the file has the same name of the file in the dropbox path, the upload won't be completed and the existing won't be overwritten.